src/styles/index.scss

```diff
@@ -1,3 +1,27 @@
-body {
+@import './utils/variable';
+@import './utils/mixins';
+@import './blocks/reset';
+@import './blocks/card';
+@import './blocks/stars';
+
+* {
   margin: 0;
 }
+
+.roboto-regular {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
```

Comment:
The font of the normal style is by default, it does not need to be specified manually.